lua/starfall/libs_cl/convar.lua

```diff
@@ -31,7 +31,7 @@ end
 function convar_library.exists(name)
 	checkpermission(instance, nil, "convar")
 	checkluatype(name, TYPE_STRING)
-	return ConVarExists(name)
+	return ConVarExists(name) and GetConVar(name)
```

Comment:
Why are you returning the convar object?

Reply:
Oh, it should be `ConVarExists(name) and GetConVar(name)~=nil`

Reply:
It won't return the convar, but a boolean when GetConVar is not nil.
ConVarExists breaks for stuff like rcon_password. It returns true but GetConVar returns nil for it.

Reply:
Can probably probably remove the ConVarExists

Reply:
Well true, both are not needed. Will make it an if statement with only GetConVar, this was probably done our of my own unseen laziness :D 
~20min

Reply:
`ConVarExists(name) and GetConVar(name)` returns the convar, not a boolean.

Reply:
Lol, nevermind me D: burn the evidence so Sevii doesn't see

Reply:
```
> print(1 and 2 and 3)
3
> print(1 or 2 or 3)
1
```